student_explorer/urls.py

```diff
@@ -30,8 +30,10 @@
 ]
 
 if 'djangosaml2' in settings.INSTALLED_APPS:
+    from djangosaml2.views import echo_attributes
     urlpatterns += (
         url(r'^accounts/', include('djangosaml2.urls')),
+        url(r'^accounts/echo_attributes/', echo_attributes),
```

Comment:
Do we need to add an overriding `accounts/logout` url like MyLA has?

https://github.com/tl-its-umich-edu/my-learning-analytics/blob/47a9d12db7a78cc099deed0e519d1b143a4cf05a/dashboard/urls.py#L77

Reply:
Yeah, it had an overriding logout but
1) It needs to be added before the djangosaml logout otherwise this one take priority
2) It needs to catch the trailing slash and not the trailing slash

I added that as a recent commit and looks like it works on dev.